elb_access_logs_bucket/main.tf

```diff
@@ -47,72 +48,81 @@ resource "aws_s3_bucket" "logs" {
       "Sid": "Stmt1503676946489",
       "Effect": "Allow",
       "Principal": {
-        "AWS": "arn:aws:iam::${lookup(var.elb_account_ids, var.region)}:root"
+        "AWS": "arn:aws:iam::${var.elb_account_ids[var.region]}:root"
```

Comment:
If I'm reading this right, we no longer need to use `lookup()` to refer to map lookups? Is that documented somewhere in upgrade guides? I was having trouble finding it.